### PR TITLE
Handle controlresults accumulation

### PR DESCRIPTION
**Pull request by Garima-msft**
## Description
</br>

- CA upc to not store RTF locally 
- after every upc checkpoint:

1.  Append csv
2.  collate scan summary 
3.  collate bug summary
4.  collate bug controlresults to be resolved 
5.  clear control results accumulation 

## Checklist
- [ ] I have read the instructions mentioned in [Contribute to Code](/Contributing.md).
- [ ] I have read and understood the criteria described under [submitting changes](/Contributing.md#submitting-changes). 
- [ ] The title of the PR clearly describes the intent of the PR.
- [ ] This PR does not introduce any breaking changes to the code.

### CODE REVIEW

File: src/AzSK.ADO/Framework/Listeners/UserReports/WriteSummaryFile.ps1 [hunk 1]
```diff
@@ -56,14 +57,17 @@ class WriteSummaryFile: FileOutputBase
 			}
 
 			# Export CSV Report
-			try 
+			if(-not [PartialScanManager]::CSVCheckpointUpdate)
```
Comment (Garima-msft) [inline src/AzSK.ADO/Framework/Listeners/UserReports/WriteSummaryFile.ps1:60]: DontWriteCSVatend

rename appropriate

File: src/AzSK.ADO/Framework/Managers/PartialScanManager.ps1 [hunk 2]
```diff
@@ -101,6 +107,8 @@ class PartialScanManager
 
 		If ($this.scanSource -eq "CA") # use storage in ADOScannerRG in case of CA scan
```
Comment (Garima-msft) [inline src/AzSK.ADO/Framework/Managers/PartialScanManager.ps1:108]: if

File: src/AzSK.ADO/Framework/Managers/PartialScanManager.ps1 [hunk 3]
```diff
@@ -15,7 +15,13 @@ class PartialScanManager
 	hidden [bool] $isRTFAlreadyAvailable = $false;
 	hidden [bool] $isDurableStorageFound = $false;
 	hidden [string] $masterFilePath;
-    $storageContext = $null;
+	$storageContext = $null;
```
Comment (Garima-msft) [inline src/AzSK.ADO/Framework/Managers/PartialScanManager.ps1:18]: Camelcase class level variables

File: src/AzSK.ADO/Framework/Managers/PartialScanManager.ps1 [hunk 4]
```diff
@@ -518,4 +554,215 @@ class PartialScanManager
 			return $false
 		}
 	}
+
```
Comment (Garima-msft) [inline src/AzSK.ADO/Framework/Managers/PartialScanManager.ps1:557]: add comments and purpose

**Review by abhaydaga — APPROVED** [review]

**Pull request merged**